crates/cheatnet/src/cheatcodes/mock_call.rs

```diff
@@ -1,15 +1,23 @@
 use crate::CheatnetState;
-use starknet_api::core::{ContractAddress, EntryPointSelector};
+use blockifier::abi::abi_utils::selector_from_name;
+use blockifier::execution::execution_utils::felt_to_stark_felt;
+use cairo_felt::Felt252;
+use cairo_lang_runner::short_string::as_cairo_short_string;
+use starknet_api::core::ContractAddress;
 use starknet_api::hash::StarkFelt;
 use std::collections::HashMap;
 
 impl CheatnetState {
     pub fn start_mock_call(
         &mut self,
         contract_address: ContractAddress,
-        function_name: EntryPointSelector,
-        ret_data: Vec<StarkFelt>,
+        function_name: &Felt252,
+        ret_data: &[Felt252],
```

Comment:
We have to set up some guides on what we want to pass to cheatcodes

Reply:
#506